static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -2509,6 +2530,88 @@
     function installAddons() {
       $scope.vm.installingAddons = true;
 
+      async.series([
+          postSuperUsers,
+          deleteSuperUsers,
+          getSuperUsers,
+          enableAddOns
+        ],
+        function (err) {
+          $scope.vm.installingAddons = false;
+          getMasterIntegrations({}, function () {});
+          if (err)
+            horn.error(err);
+        }
+      );
+    }
+
+    function postSuperUsers(next) {
+      if (!$scope.vm.superUsers.addedSuperUsers.length)
+        return next();
+
+      async.each($scope.vm.superUsers.addedSuperUsers,
+        function (superUserAccountId, done) {
+          var newSuperUser = {
+            accountId: superUserAccountId
+          };
+          admiralApiAdapter.postSuperUser(newSuperUser,
+            function (err) {
+              if (err)
+                return done(err);
+              return done();
+            }
+          );
+        },
+        function (err) {
+          $scope.vm.superUsers.addedSuperUsers = [];
```

Comment:
Should this be set to an empty array on an error, or should we give the user a chance to correct it?